### PR TITLE
BayDAG parking location code changes

### PR DESCRIPTION
**Pull request by AshishKuls**

### CODE REVIEW

**Comment by kulshresthaa** [comment]
@dhensle This PR is ready for your review. Thanks!

**Review by dhensle — CHANGES_REQUESTED** [review]

File: activitysim/abm/models/parking_location_choice.py [hunk 1]
```diff
@@ -259,18 +255,33 @@ def parking_location(
...
     land_use_df = land_use.to_frame()
 
-    locals_dict = {
-        'network_los': network_los
-    }
-    locals_dict.update(config.get_model_constants(model_settings))
+    alt_column_filter_name = model_settings.get('ALTERNATIVE_FILTER_COLUMN_NAME')
+    alt_tdd = land_use_df[land_use_df[alt_column_filter_name]]
+    alt_tdd[alt_destination_col_name] = alt_tdd.index
+
```
[the remark this answers is not in the document]
Comment (kulshresthaa) [reply activitysim/abm/models/parking_location_choice.py]: Do you mean `network_los` isn't needed or `skims` needs to be added in locals_dict?

File: activitysim/abm/models/parking_location_choice.py [hunk 2]
```diff
@@ -259,18 +255,33 @@ def parking_location(
...
+        expressions.assign_columns(
+            df=alt_tdd,
+            model_settings=alt_preprocessor_settings,
+            locals_dict=locals_dict,
+            trace_label=trace_label)
+
+    # trips preprocessor
+    preprocessor_settings = model_settings.get('PREPROCESSOR', None)
     if preprocessor_settings:
+        locals_dict = {}
```
[the remark this answers is not in the document]
Comment (kulshresthaa) [reply activitysim/abm/models/parking_location_choice.py]: got rid of the alts preprocssor, as it wasn't needed.

File: activitysim/abm/models/parking_location_choice.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhensle) [reply activitysim/abm/models/parking_location_choice.py]: I mean that the skims need to be in the locals_dict somehow, whether this is through the `network_los` object or the some other `skims` object doesn't really matter to me.  I suggested above just adding `network_los`.

[the remark this answers is not in the document]
Comment (dhensle) [reply activitysim/abm/models/parking_location_choice.py]: I just kept the network_los object in there upon merging.

**Pull request merged**